src/components/functional/notifications-feed/notifications-feed.tsx

```diff
@@ -58,18 +57,12 @@ export class NotificationsFeed {
     const hasPending = this.pendingTransactions?.length > 0;
 
     return (
-      <mvx-side-panel isOpen={this.isOpen} side={SidePanelSideEnum.RIGHT} onClose={this.handleClose}>
+      <mvx-side-panel isOpen={this.isOpen} panelTitle="Notifications Feed" onClose={this.handleClose}>
```

Comment:
title ?

Reply:
There's a TypeScript warning, or lint issue regarding this, apparently `title` is a reserved keyword in Stencil.
![Uploading image.png…]()

Reply:
There's a TypeScript warning, or lint issue regarding this, apparently title is a reserved keyword in Stencil.
<img width="857" alt="image" src="https://github.com/user-attachments/assets/f29b2011-67b4-481d-ab81-149678b2d2cd" />